travis/publish.sh

```diff
@@ -104,6 +104,50 @@ safe_checkout_master() {
   fi
 }
 
+test_server() {
+  # Test Stackdriver Storage module with the latest version of Zipkin server.
+  temp_dir=$(mktemp -d)
+  pushd $temp_dir
+
+  # Download and unpack Zipkin Server
+  wget https://jitpack.io/com/github/openzipkin/zipkin/zipkin-server/master-SNAPSHOT/zipkin-server-master-SNAPSHOT-exec.jar
```

Comment:
Let's use the sonatype snapshots repo instead.

Reply:
i'm having some trouble finding where this is :( a pointer would be appreciate it!

Reply:
I don't agree with using sonatype as it implies maintenance to get the latest snapshot version. Let's only do that if we must.

Reply:
for example, it looks like this, which is unnecessary blinding

```
    # the current version of zipkin-lens is in a path of the same name in a jar file. This extracts it.
    LENS_BASE=$ZIPKIN_REPO/io/zipkin/zipkin-lens && \
    LENS_VERSION=$(curl -sSL $LENS_BASE/maven-metadata.xml | sed -n '/<version>/s/.*<version>\([^<]*\)<\/version>.*/\1/p'|tail -1) && \
    LENS_BUILD=$(curl -sSL $LENS_BASE/$LENS_VERSION/maven-metadata.xml | sed -n '/<value>/s/.*<value>\([^<]*\)<\/value>.*/\1/p'| head -1) && \
    curl -sSL $LENS_BASE/$LENS_VERSION/zipkin-lens-$LENS_BUILD.jar > zipkin-lens.jar && \
```

https://github.com/openzipkin/docker-zipkin/blob/master/zipkin-ui/Dockerfile#L10-L14

If we end up with problems in jitpack, we can revisit, but please let's keep as much cruft out as possible.

Reply:
We've already had problems with running the wrong version of a jitpack build without specifying a hash. The caching behavior is too opaque to put in an automated test IMO.

Reply:
I'm not positive that was jitpack's fault. you and I were flipping between many things and with not much rest. It is entirely possible the flipping with no sleep was more an issue than jitpack iotw

Reply:
ps either way translates to the same.. if someone was too tired and this didn't work or similar because they didn't force docker to invalidate cache etc.. I'm fine removing jitpack from this step if you feel strongly about it. we will still have the docker cache miss thing on any snapshot though.. even what I pasted scraping xml.

tldr I'm ok either way on this part

Reply:
tabled as the shell scripting to do the alternative is gross #155